app/Http/Controllers/ShopController.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Product;
+use Illuminate\Http\Request;
+use Inertia\Inertia;
+
+class ShopController extends Controller
+{
+    public function show(Request $request)
+    {
+        $edition = $request->input('edition');
+
+        if ($edition === null) {
+            return response('No edition found', 500);
+        }
+
+        $products = Product::where('edition_id', $edition->id)->get();
```

Comment:
```suggestion
        $products = $edition->products;
```

This is more idiomatic, in my opinion.

Reply:
You could even inline this in the return value